### PR TITLE
[#2901] contacts on conversation view

### LINKED ISSUE #2901: Contacts on Conversation view
Contacts feature is OFF by default. This ticket is scoped on _Contacts feature = ON._ 

**CONTACTS = OFF** 

User story: (same as current Airy Version 0.41.0)

1. User agent receives a new message on Airy UI.
2. User agent opens the new message, one can see conversation.
3. On third column, User agent is able to see:
       3.1. Avatar = conversation.list : "image_url"
       3.2. Sender Name = "conversation.list" : "display_name"
       3.3. Tags 
4. User agent replies to the message.

**CONTACTS = ON**

### **"USER AGENT " general success scenarios :** 

1. User agent receives a new message on Airy UI.
2. User agent opens the new message, one can see conversation & _Contacts information_ on the third column of the view.
       2.1. Before user agent replies to the customer, one can see the following;
       2.2. Contact Name, So user agent can start with a personalised welcome message using the 
               contact name.
       2.3 Contact email, if exists. User agent can start sending email to contact : "Mailto"
       2.4 Contact phone numbers; if exists.
       2.5 Title; if exists, 
       2.6 Address; if exists, 
       2.7 Organisation; if exists, 
4. User agent replies to the message with all the previous information on their screen to assist this contact.

### **FRONTEND WORK** 

**Task (1) : Show contacts in conversation level on third column**  
  _Explanation:_ " Pass "contact.list" datapoints (mentioned below) to Contact component on frontend:
                             - Avatar: “contact.create” = “{avatar_url:”source.messenger.avatar.url”}”
                             - Name: “{“display_name”:”source.sender.name”}”
                             - Email: “contact.create” = {“via” : “email”}
                             - Phone: “contact.create” = {“via” : “phone”}
                             - Title: "contact.create” = “{“title”:”null”}
                             - Address: ”contact.create” = “{“address” : “address.line1” + “city”}”
                             - Organisation: ”contact.create” = {“address” : “organisation_name”}

**Task (2) : Edit Functionality** 
  _Explanation:_ 
- Button:  “ EDIT symbol” : "view.contact.component"
       - ACTION: Hide edit button **&** replace with “exit” symbol” = exist edit mood.
       - ACTION : Edit = "contacts.data.points"
       - ACTION: “Save button” = contact.update

**Task (3) : Component behaviour** 
  _Explanation:_
- Button: See more (x)
      - ACTION: expand all contacts.data.points are visible.
- Button: See less 
      - ACTION: collapse contact.data.points. Hide last TWO data points.  

FIGMA Link: https://www.figma.com/file/aS4AO0yX3dMonNskWD6D77/Contact-Conversation-Level?node-id=0%3A1

### PR DESCRIPTION
**Pull request by AudreyKj**
closes #2901

### CODE REVIEW

**Review by chrismatix — COMMENTED** [review]
Looks very good already. There's one general question that we need to think about though: Like the `media-resolver` contacts can be turned off or just unavailable. 

Do you think it'd be reasonable to wrap your code in a check like an upload functionality?

File: WORKSPACE [hunk 1]
```diff
@@ -49,6 +49,17 @@ http_archive(
...
+    strip_prefix = "zlib-1.2.11",
+    urls = [
+        "https://mirror.bazel.build/zlib.net/zlib-1.2.11.tar.gz",
+        "https://zlib.net/zlib-1.2.11.tar.gz",
+    ],
+)
+
 load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
 
 go_rules_dependencies()
```
Comment (AudreyKj) [inline WORKSPACE:65]: linting fix for the CI

File: frontend/ui/src/pages/Inbox/Messenger/ConversationMetadata/ContactDetails/ContactInfoPoint.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {ReactComponent as Email} from 'assets/images/icons/email.svg';
```
Comment (thorstenairy) [inline frontend/ui/src/pages/Inbox/Messenger/ConversationMetadata/ContactDetails/ContactInfoPoint.tsx:2]: nitpick: as we name those icons with an additional Icon (EmailIcon) so it's more clear that this component "just" a svg.

**Review by thorstenairy — APPROVED** [review]
LGTM 🚀

**Pull request merged**